style.css

```diff
@@ -658,105 +715,443 @@ body {
+  }
+
+  #m {
+    margin-top: 0%;
+    margin-bottom: 0%;
+    padding: 0%;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  #description {
+    width: 100%;
+    margin-top: 25px;
+    padding: 0% 0%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .box {
+    padding: 0% 0%;
+    margin: 0% 0%;
+  }
+
+  .apps {
+    padding: 0% 0%;
+    margin: 0% 0%;
+  }
+
+  #btn_1 {
+    border: none;
+    margin: 10px 10px;
+  }
+
+  #mob_ver {
+    display: none;
+  }
+
+  .ex1 {
+    display: flex;
+    background: url(images/sada.png);
+    background-size: cover;
+    background-position: center;
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 2%;
+    margin-top: 2%;
+  }
+
+  .ex2 {
+    display: flex;
+    background: url(images/exc.png);
+    background-size: cover;
+    background-position: center;
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 20px;
+    margin-top: 20px;
+  }
+
+  .ex3 {
+    display: flex;
+    background: url(images/sada1.png);
+    background-size: cover;
+    background-position: center;
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 20px;
+    margin-top: 20px;
+  }
+
+  .ex4 {
+    display: flex;
+    background: url(images/sada2.png);
+    background-size: cover;
+    background-position: center;
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 20px;
+    margin-top: 20px;
+    justify-content: flex-end;
+  }
+
+  #r1 {
+    display: flex;
+  }
+
+  #desk_ver {
+    width: 85%;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 10vh;
+  }
+
+  .about_myself {
+    background-image: url(images/abt.png);
+    background-repeat: no-repeat;
+    background-origin: content-box;
+    background-position: 100%;
+    background-attachment: scroll;
+    background-size: 500px;
+    display: flex;
+    flex-direction: column;
+    background-color: #262626;
+    margin-bottom: 0%;
+    padding-bottom: 0%;
+  }
```

Comment:
As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 38 15 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/062a7a31-d6be-498c-b521-3d47e0ec50b3">
>
> ## Your Design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 13 22 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/36dbf1e7-5ebe-4704-9e40-622ddc5064e9">
><img width="1440" alt="Screenshot 2023-06-15 at 11 16 16 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/998bf2f5-2c0f-44ea-b92d-4d1e940c4c02">